### DISCUSSION TITLE
Unable to login or sign up

### DISCUSSION THREAD

**Discussion by myothercoolname** [vercel/community#774] [Help] [answered]
hi 👋 

About 2 years ago I signed up to vercel with a (different) github account. When I try to log in now I get this error:
"There is no Vercel account linked with this GitHub account."

I tried to create a new vercel account but I got stuck because when I enter my phone number for verification I get this error:
"This phone number is already associated with a Vercel account."

I checked vercel's support page but in order to create a case I have to be logged in... so I'm here now :) 

Could someone please help me?

Thanks!

**Comment by swarnava** [comment, marked as answer]
Can you reach out to us using your registered email and Phone number so that we can take a look? registration@vercel.com
